Dockerfile

```diff
@@ -0,0 +1,31 @@
+FROM golang:alpine as BUILD
+
+WORKDIR /relayer
+
+# Copy the files from host
+COPY . .
+
+# Update and install needed deps prioir to installing the binary.
+RUN apk update && \
+    apk --no-cache add make=4.2.1-r2 git=2.24.3-r0 && \
```

Comment:
do we need to specify `make` and `git` `rc` versions here?

Reply:
we can go without, I was experimenting with a new tool (https://github.com/hadolint/hadolint) and it recommended it.